recipes/gw/meta.yaml

```diff
@@ -68,6 +75,8 @@ about:
   summary: 'View genomic sequencing data and vcf files'
 
 extra:
+  additional-platforms:
+    - osx-arm64
```

Comment:
How about adding `linux-aarch64` too ? Would it work ?

Reply:
Not yet I'm afraid. I will add this soon though!